### ISSUE TITLE
Cannot read property 'length' of null - Can't login

### ISSUE THREAD

**Issue by bscott** [johnjones4/Standard-Notes-Clipper#30]
After Trying to log in and asked for my 2FA code, I receive the following error:

`Cannot read property 'length' of null`

Using Edge browser

Console Log:

```
settings.js:1529 TypeError: Cannot read property 'length' of null
    at SFCryptoWeb._callee128$ (chrome-extension://fihpcbcgcjejccokoekllnnafdilelae/background/background.js:34284)
    at tryCatch (chrome-extension://fihpcbcgcjejccokoekllnnafdilelae/background/background.js:42247)
    at Generator.invoke [as _invoke] (chrome-extension://fihpcbcgcjejccokoekllnnafdilelae/background/background.js:42473)
    at Generator.prototype.<computed> [as next] (chrome-extension://fihpcbcgcjejccokoekllnnafdilelae/background/background.js:42299)
    at asyncGeneratorStep (chrome-extension://fihpcbcgcjejccokoekllnnafdilelae/background/background.js:23836)
    at _next (chrome-extension://fihpcbcgcjejccokoekllnnafdilelae/background/background.js:23838)
```

**Comment by bscott** [comment]
@johnjones4 if you want to add me as a Maintainer, I wouldn't mind working on some of the issues and accepting PR's

**Comment by johnjones4** [comment]
Absolutely! Sorry I've been flaking on this. A few too many pots on the stove right now.

**Comment by bscott** [comment]
@johnjones4 Np!, Thanks for adding me and allowing me to help :)

**Comment by ThePauler** [comment]
Same issue on firefox and edge.  Try to log in and I get 
`Cannot read property 'length' of null`

Hope it can get fixed.  Would love to use this app for web clipping

**Comment by anacoluthn** [comment]
Having the same issue here on Chrome, hope it can get fixed soon!

**Comment by bscott** [comment]
@anacoluthn working on it, hoping to have a minor bug fix release this weekend

**Comment by bscott** [comment]
Update:

The Javascript library used in the SN Web Clipper is no longer supported by Standard Notes, That package is obsolete. The new package is SNJS. I am working on doing a refactor using the new SN JS library. We Will need about a week to have this done then will need to test and create a new release.

**Comment by Librechain** [comment]
Hey, 

Just commenting so I can get an alert on this issue when the patch drops as I'm having the same issue as the users above. 

Great job @bscott + @johnjones4 ; thanks a load for helping the community out. I'm grateful.

**Comment by bscott** [comment]
Thanks, working thru this week and testing with a few folks next week before releasing.

**Comment by bscott** [comment]
@johnjones4 Can you email me at `bscott at hey.com`, need to know your last release process, I have refactored using the new SNJS library.

**Comment by johnjones4** [comment]
Thanks so much for your efforts here! I just emailed you

**Comment by cherya** [comment]
@bscott any updates on it? Maybe you need some help with testing or something?

**Comment by bscott** [comment]
Hi

I'm preparing a beta release for testing, resolving a few bugs first.

**Comment by ThePauler** [comment]
Any updates?  Need beta testers?

**Comment by bscott** [comment]
@ThePauler Yes, a few minor issues I need to fix, looking for any contributors to speed up the release.
